transport/tchannel/handler.go

```diff
@@ -108,33 +121,34 @@ func (h handler) handle(ctx context.Context, call inboundCall) {
 		call.Response().Blackhole()
 		return
 	}
-	if err != nil && !responseWriter.isApplicationError {
-		// TODO: log error
+	if err != nil && !responseWriter.IsApplicationError() {
+
 		_ = call.Response().SendSystemError(getSystemError(err))
+		h.logger.Error("tchannel callHandler error", zap.Error(err))
```

Comment:
"callHandler error" is a bit generic. This could be more descriptive, like "tchannel send system error."